### PR TITLE
Add new sentry setup

### PR DESCRIPTION
**Pull request by yosifkit**
Docs for https://github.com/docker-library/sentry/pull/15

### CODE REVIEW

File: sentry/content.md [hunk 1]
```diff
@@ -8,19 +8,41 @@ Sentry is a realtime event logging and aggregation platform. It specializes in m
...
+		docker run -d --name some-redis redis
 
-### MySQL database
+2.	start a MySQL or PostgreSQL(recommended by upstream) database container (later steps assume PostgreSQL, replace the `--link` entries appropriately for MySQL)
 
-	docker run --name some-sentry --link some-mysql:mysql -d sentry
+		# postgres:
+		docker run -d -e POSTGRES_PASSWORD=secret -e POSTGRES_USER=sentry --name some-postgres postgres
+		# or mysql:
+		docker run -d -e MYSQL_ROOT_PASSWORD=secret -e MYSQL_DATABASE=sentry --name some-mysql mysql
```
Comment (tianon) [inline sentry/content.md]: If you change these to a sub-list, you can make this slightly easier to look at: (and the intent of that initial sentence easier to grok, especially at a glance)
```markdown
2.	start a database container

	-	PostgreSQL: (recommended by upstream)

			docker run -d -e POSTGRES_PASSWORD=secret -e POSTGRES_USER=sentry --name some-postgres postgres

	-	MySQL: (later steps assume PostgreSQL, replace the later `--link` entries appropriately for MySQL)

			docker run -d -e MYSQL_ROOT_PASSWORD=secret -e MYSQL_DATABASE=sentry --name some-mysql mysql
```

Rendered:

2.	start a database container

	-	PostgreSQL: (recommended by upstream)

			docker run -d -e POSTGRES_PASSWORD=secret -e POSTGRES_USER=sentry --name some-postgres postgres

	-	MySQL: (later steps assume PostgreSQL, replace the later `--link` entries appropriately for MySQL)

			docker run -d -e MYSQL_ROOT_PASSWORD=secret -e MYSQL_DATABASE=sentry --name some-mysql mysql

File: sentry/content.md [hunk 2]
```diff
@@ -8,19 +8,41 @@ Sentry is a realtime event logging and aggregation platform. It specializes in m
...
 
-### Redis buffering (recommended by upstream for any real workloads)
+3.	now start up sentry server
 
-To enable Update Buffers using Redis, just add `--link some-redis:redis` to the `docker run` arguments of your service.
+		docker run -d --name some-sentry --link some-redis:redis --link some-postgres:postgres sentry
+		# or
+		docker run -d --name some-sentry --link some-redis:redis --link some-mysql:mysql sentry
```
Comment (tianon) [inline sentry/content.md]: You already told me these steps would assume `postgres`, so I'd say let's axe this second option for simplicity.

File: sentry/content.md (hunk 1 above)
Comment (tianon) [inline sentry/content.md]: Probably change that note to explicitly say something more like "replace the later `--link some-postgres:postgres` with `--link some-mysql:mysql`".

**Comment by yosifkit** [comment]
Done, PTAL :pray:.

**Comment by tianon** [comment]
LGTM (pending https://github.com/docker-library/official-images/pull/919)

**Pull request merged**